.github/workflows/solidity.yml

```diff
@@ -26,15 +26,14 @@ jobs:
               - 'contracts/**/*'
               - '.github/workflows/solidity.yml'
               - '.github/workflows/solidity-foundry.yml'
-            old_sol:
```

Comment:
Afaik @smartcontractkit/keepers has some frozen code that should be included here so I'll keep this logic around. I've added the deprecated interface folder here for now, as no files should be added there